CHANGELOG.md

```diff
@@ -18,6 +18,21 @@ Use the following schema when setting up the Changelog for a new release. Remove
 ### Security
 -->
 
+## [Unreleased]
+
+### Added
+
+### Changed
+
+### Deprecated
+
+### Removed
+- Dependency on `nscala-time`, using only `joda.time` methods instead ([#849](https://github.com/adzerk/apso/pull/849)).
```

Comment:
It's OK to not eagerly update the Changelog if you don't feel like it. We usually run something like the following prior to a release to figure out which items to include.

```sh
gh pr list --limit 100 --state merged --search "base:master merged:>2025-04-07T10:09:18+0100" --json title,url,number --template '{{range .}}- {{.title}} ([#{{.number}}]({{.url}})).{{"\n"}}{{end}}' | tac
```